step-capstone/src/components/Trip.js

```diff
@@ -12,99 +12,85 @@ const testData = [
         finalized: true,
         type: "flight",
         departureAirport: "BOS",
+        departureCoordinates: {
+            lat: 42.365360,
+            lng: -71.008187
+        },
         arrivalAirport: "SFO",
+        arrivalCoordinates: {
+            lat: 51.514212,
+            lng: 0.057513
+        },
         departureDate: new Date(),
         arrivalDate: new Date(),
         description: "Additional notes"
     },
     {
-        id: 1,
-        finalized: false,
-        type: "flight",
-        departureAirport: "BOS",
-        arrivalAirport: "SFO",
-        departureDate: new Date(),
-        arrivalDate: new Date(),
-        description: "Additional notes"
-    },
-    {
-        id: 2,
-        finalized: false,
-        type: "flight",
-        departureAirport: "BOS",
-        arrivalAirport: "SFO",
-        departureDate: new Date(),
-        arrivalDate: new Date(),
-        description: "Additional notes"
-    },
-    {
-        id: 3,
-        finalized: false,
-        type: "flight",
-        departureAirport: "BOS",
-        arrivalAirport: "SFO",
-        departureDate: new Date(),
-        arrivalDate: new Date(),
-        description: "Additional notes"
-    },
-    {
-        id: 125,
-        finalized: true,
-        type: "flight",
-        departureAirport: "BOS",
-        arrivalAirport: "SFO",
-        departureDate: new Date(),
-        arrivalDate: new Date(),
-        description: "Additional notes"
-    },
-    {
-        id: 87,
+        id: 100,
         finalized: true,
-        type: "flight",
-        departureAirport: "BOS",
-        arrivalAirport: "SFO",
-        departureDate: new Date(),
-        arrivalDate: new Date(),
-        description: "Additional notes"
+        type: "hotel",
+        title: "The Goring",
+        location: "15 Beeston Pl, Westminster, London SW1W 0JW, United Kingdom",
+        coordinates: {
+            lat: 51.497735,
+            lng: -0.145621
+        },
+        startDate: new Date(),
+        endDate: new Date(),
+        description: 'description',
     },
     {
-        id: 231,
-        finalized: true,
-        type: "flight",
-        departureAirport: "BOS",
-        arrivalAirport: "SFO",
-        departureDate: new Date(),
-        arrivalDate: new Date(),
-        description: "Additional notes"
+        id: 327,
+        finalized: false,
+        type: "hotel",
```

Comment:
Shouldn't this be an event, not a hotel? Is event handling not ready yet? Applies to the below as well.

Reply:
I think Event component hasn't been merged yet. Having an event object is fine, but won't do anything right now so I only tested with hotels which have the exact same properties as events.